doc/pachctl/pachctl_port-forward.md

```diff
@@ -14,12 +14,12 @@ Forward a port on the local machine to pachd. This command blocks.
 ### Options
 
 ```
-  -k, --kubectlflags string   Any kubectl flags to proxy, e.g. --kubectlflags='--kubeconfig /some/path/kubeconfig'
-      --namespace string      Kubernetes namespace Pachyderm is deployed in. (default "default")
-  -p, --port int              The local port to bind pachd to. (default 30650)
-  -x, --proxy-port int        The local port to bind Pachyderm's dash proxy service to. (default 30081)
-      --saml-port int         The local port to bind pachd's SAML ACS to. (default 30654)
-  -u, --ui-port int           The local port to bind Pachyderm's dash service to. (default 30080)
+      --namespace string   Kubernetes namespace Pachyderm is deployed in. (default "default")
```

Comment:
Can we have -n also work for --namespace like k8s does?

Reply:
https://github.com/pachyderm/pachyderm/issues/3395